release notes/v0.44.0.md

```diff
@@ -0,0 +1,223 @@
+k6 v0.44.0 is here 🎉! This release includes:
+
+- A new experimental Web Crypto module.
+- A sampling option for the experimental tracing module.
+- A memory usage improvements.
+- Bug fixes and UX improvements.
+
+Some highlights from the k6-browser module are:
```

Comment:
```suggestion
Some highlights from the browser module are:
```
I think we should be consistent. We use all the combinations (xk6-browser, k6 browser, k6-browser).  I'm for referring to it as just `browser module` now that we merged the extension. `k6 browser` is also fine to me but it sounds not required repetition. Of course, it does not include the links to the extension repository. WDYT? @grafana/k6-browser @olegbespalov

Reply:
I think there was a discussion around the naming of this somewhere internally, so I believe the browser's team knows better

Reply:
We recently decided to start calling it `k6 browser` instead of `xk6-browser` from our side, but I think from the k6 perspective it makes sense to refer to it as `browser module`, that's also how it's called in the [documentation](https://k6.io/docs/javascript-api/k6-experimental/browser/):

> The browser module APIs aim for rough compatibility with the [Playwright API for NodeJS](https://playwright.dev/docs/api/class-playwright).

Reply:
That means that we could probably use something like:

```diff
-[xk6-browser#790](https://github.com/grafana/xk6-browser/pull/790)
+[browser#790](https://github.com/grafana/xk6-browser/pull/790)
```

when we refer to the PR or the Issue in the browser's repo. And should not use the `xk6-browser` in texting:

```diff
-### Web Vitals are now natively supported by the xk6-browser module
+### Web Vitals are now natively supported by the k6 browser module
```

 Right?

@ankur22 will you take care of it?

Reply:
Yeah, makes sense. I've renamed from k6-browser/k6 browser to browser module. There are two cases which still feel valid as they reference the xk6-browser repo (update to a readme and the binary). change is in c61f4f03a5dde736872430e3637c4a2d7606fcef

Reply:
@olegbespalov I think it's worth keeping it as `xk6-browser` since it's a reference to the repo itself, WDYT?

Reply:
@grafana/k6-browser should we do this change too. I don't have a strong opinion on either:

```suggestion
-[xk6-browser#790](https://github.com/grafana/xk6-browser/pull/790)
+[browser#790](https://github.com/grafana/xk6-browser/pull/790)
```

Reply:
Personally I prefer to change it and stick to `browser` module naming for any reference done to it from `k6`. For me `xk6-*` is a complicate naming, and I think it makes more sense to speak just about `browser`. The fact that the PR link will lead you to the `xk6-browser` repo and the naming mismatch is a minor "issue" for me, if it can be considered an issue :man_shrugging:

Reply:
Sounds good, changed in 1375a51c50f5b1e883d1b9c348d0a1ed754c00fc